README.md

```diff
@@ -90,6 +104,3 @@ This config creates a VM with this software:
 To run a deploy script:
```

Comment:
Actually we could now delete this section (Running deploy scripts) because the only script that we have is for the dbs and you documented it above. Could you delete it? :grin: